Source/WebKit/Scripts/generate-serializers.py

```diff
@@ -469,6 +509,17 @@ def generate_impl(serialized_types, serialized_enums, headers):
     result.append('#include "config.h"')
     result.append('#include "GeneratedSerializers.h"')
     result.append('')
+    result.append('template<size_t...> struct MembersInCorrectOrder;')
+    result.append('template<size_t onlyOffset> struct MembersInCorrectOrder<onlyOffset> { static constexpr bool value = true; };')
+    result.append('template<size_t firstOffset, size_t secondOffset, size_t... remainingOffsets> struct MembersInCorrectOrder<firstOffset, secondOffset, remainingOffsets...> {')
+    result.append('    static constexpr bool value = firstOffset > secondOffset ? false : MembersInCorrectOrder<secondOffset, remainingOffsets...>::value;')
```

Comment:
Discussed this offline with Alex and we are using `>` instead of `>=` here to support things like serializing zero-sized members, e.g. those marked with `[[no_unique_address]]`.